PopulationLevelEstimation.Rmd

```diff
@@ -320,7 +320,7 @@ The option "**Should only the first exposure per subject be included?**" can be
 - "**Keep First**" indicating to keep the subject in the first cohort that occurred.
 - "**Remove All**" indicating to remove the subject from both cohorts.
 
-If the options "keep all" or "keep first" are selected, we may wish to censor the time when a person is in both cohorts. This is illustrated in Figure \@ref(fig:tar). By default, the time-at-risk is defined relative to the cohort start and end date. In this example, the time-at-risk starts one day after cohort entry, and stops at cohort end. Without censoring the time-at-risk for the two cohorts might overlap. This is especially problematic if we choose to keep all, because any outcome that occurs during this overlap (as shown) will be counted twice. If we choose to censor, the first cohort's time-at-risk ends when the second cohort's time-at-risk starts.
+If the options "keep all" or "keep first" are selected, we may wish to censor the time when a person is in both cohorts. This is illustrated in Figure \@ref(fig:tar). By default, the time-at-risk is defined relative to the cohort start and end date. In this example, the time-at-risk starts one day after cohort entry, and stops at cohort end. Without censoring the time-at-risk for the two cohorts might overlap. This is especially problematic if we choose to keep all, because any outcome that occurs during this overlap (as shown) will be counted twice. If we choose to censor (i.e., choose "Keep First"), the first cohort's time-at-risk ends when the second cohort's time-at-risk starts.
```

Comment:
Actually, whether we censor or not depends on the option *"If a subject is in multiple cohorts, should time-at-risk be censored when the new time-at-risk starts to prevent overlap?”*, not on whether we choose "Keep first".

Reply:
Oh! I see! The words belongs to the "If a subject is in ... overlap?”. Thanks !!